src/plugin/file/zip/zipdescriptor.js

```diff
@@ -0,0 +1,47 @@
+goog.provide('plugin.file.zip.ZIPDescriptor');
```

Comment:
Unless you plan on showing the ZIP file in Add Data, you don't need to create a descriptor for the overall ZIP. We're creating individual descriptors for each file imported from the ZIP, so this one should not be necessary.

Reply:
Removed!